dashboard/templates/registration/login.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'base.html' %}
+
+{% block title %}Login{% endblock %}
```

Comment:
is the login link integrated with Shib?

Reply:
Yeah, this login.html page is just used for non shibboleth login. The link is in the base.html